### ISSUE TITLE
About screen compose

### ISSUE THREAD

**Issue by lingboo** [ThorGroup/FinalProject#12]
@ilentt i think, you need to change position of layout. I think button post should be at toolbar and replace with an icon, and icon category in right top at  toolbar should be at bottom, and use EditText.
It's like following picture: [http://imgur.com/Y5C93ex](url)

**Comment by ilentt** [comment]
Ok @lingbo I wil move post button go on top inline with camera and image
button. Category button I will move into edit text box like this

[image: Hình ảnh nội tuyến 1]

2016-04-25 18:51 GMT+07:00 lingboo <notifications@github.com>:

**Comment by lingboo** [comment]
I can't see the picture, please check again

**Comment by ilentt** [comment]
Go into your  inbox, I think you can see it. If not I will update link
again.

2016-04-25 21:29 GMT+07:00 lingboo <notifications@github.com>:

**Comment by ilentt** [comment]
@lingboo I have changed location classify and add more submit button on the top bar(icon hand), you need implement click action again. I have add normal feeling option as we discuss on Sunday too. About crop image I need ask teacher some technical issues I am going to complete at class today.

@perich On last Sunday me and @lingboo decided add once more option for reaction feature It means we have three option instead of two option in current. Please add once more attribute on your back end for this feature, ask @lingboo  if you need explain more detail.

**Comment by perich** [comment]
@lingboo @ilentt What else did you add? I should just be true/false or yes/no. What is the 3rd reaction choice?

**Comment by ilentt** [comment]
The first reaction just store 2 value true/false or yes/no, and now you
just change data type such as from boolean(true, false) to integer for 3
value(1, 2, 3)

2016-04-26 8:57 GMT+07:00 Peter Richards <notifications@github.com>:

**Comment by perich** [comment]
@ilentt @lingboo No, it's just two values "Like" and "Dont like". It's a boolean

**Comment by ilentt** [comment]
why do you can not change the data type. Is it very complicated right? or
some else reason?

2016-04-26 13:55 GMT+07:00 Peter Richards <notifications@github.com>:

**Comment by perich** [comment]
@ilentt We dont need a 3rd type. Its just like/dont like. Why do we need a 3rd type? It has been boolean since beginning. Everybody code it as boolean. Why you want to change it now?

**Comment by ilentt** [comment]
Two options(like or don't like) are pretty good still now. However when I
work with @lingbo he suggest we should have 3 options such as boring, fine
and exciting. I think it is a good idea, it will impressed the judges and I
agree with him. So if it is not complicated we should vote for this idea.

How do you think @tanlnm512, @lingbo

2016-04-26 14:06 GMT+07:00 Peter Richards <notifications@github.com>:

**Comment by perich** [comment]
@ilentt @lingboo We cannot go making changes like this a few days before this is supposed to be finished. You also didnt talk to the group about this. Of course that it more work to implement on the backend. I vote definitely against this. I dont think having 2 vs 3 options will impress anyone. But if you really want to change it, you're welcome to.

**Comment by ilentt** [comment]
I vote acceptance 3 options if it not complicated.

2016-04-26 14:37 GMT+07:00 Peter Richards <notifications@github.com>:

**Comment by perich** [comment]
@ilentt Please join facebook chat so we can figure this out.

**Comment by ilentt** [comment]
Sorry @perich I can not join FB in working times. You should discuss
together and I will accept the last result.

2016-04-26 14:58 GMT+07:00 Peter Richards <notifications@github.com>:

**Comment by perich** [comment]
@ilentt Ok, then I think we cannot do this. We dont have time to discuss, we dont have time to implement. Lets just finish what we have before we start new stuff.

Also, I dont see your compose stuff, what branch is it on? I want to finsh my part, but I need your compose feature to do it.

**Comment by ilentt** [comment]
Ok @perich I will finish my task before start new stuff.

2016-04-26 15:07 GMT+07:00 Peter Richards <notifications@github.com>:
